### ISSUE TITLE
Add bullet in PR checklist to acknowledge reviewers of blogposts

### ISSUE THREAD

**Issue by jamesmbaazam** [epiverse-trace/epiverse-trace.github.io#146]
I just noticed a mixed practice of acknowledging reviewers of our blog posts. I would suggest adding a bullet to the section on "Right before merging" in the [pull request template](https://github.com/epiverse-trace/epiverse-trace.github.io/blob/main/.github/pull_request_template.md) to remind authors to acknowledge reviewers for their effort.

**Comment by Bisaloo** [comment]
Yes, this sounds like a good idea to me. Could you submit a PR please?
